db/schema.rb

```diff
@@ -50,6 +47,7 @@
     t.boolean  "repeatable"
     t.datetime "created_at",                          null: false
     t.datetime "updated_at",                          null: false
+    t.integer  "modifiable_before",      default: 0
```

Comment:
Might have to think about the default value. 0 makes it unmodifiable by default. @jessyurko Shall we keep it 48 hours?